### PR TITLE
More logs to determine the root cause of null formController

### PR DESCRIPTION
**Pull request by grzesiek2010**
#### What has been done to verify that this works as intended?
I've just reviewed implemented changes.

#### Why is this the best possible solution? Were any other approaches considered?
Nothing to discuss here.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
It doesn't require testing.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java [hunk 1]
```diff
@@ -2105,6 +2105,8 @@ public void loadingComplete(FormLoaderTask task, FormDef formDef, String warning
         final FormController formController = task.getFormController();
 
         if (formController != null) {
+            Timber.w("Form %s started", formController.getFormTitle());
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java]: Can we hash this please? I know this kind of info sometimes shows up in crash logs but when we're explicitly logging I think we should hash.

Comment (grzesiek2010) [reply collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java]: Ok we can do that.

File: collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt [hunk 2]
```diff
@@ -49,6 +50,7 @@ class FormUriActivity : ComponentActivity() {
             logAnalytics(uriProjectId)
 
             if (projectId == currentProjectProvider.getCurrentProject().uuid) {
+                Timber.w("Form started via VIEW/EDIT actions")
```
Comment (seadowg) [inline collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt]: Are we not already logging this [through an analytics event](https://github.com/getodk/collect/blob/55c00b8ce6f2a08d94a84713362f56be4ad0a3ce/collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt#L49)? @lognaturel pointed that out to me the other day.

Comment (grzesiek2010) [reply collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt]: Hmm let me check it...

Comment (grzesiek2010) [reply collect_app/src/main/java/org/odk/collect/android/external/FormUriActivity.kt]: Right that event should be visible so we do not need a new one. Fixed.

**Review by seadowg — APPROVED** [review]

**Comment by seadowg** [comment]
@lognaturel merging this now, but take a look to check you're that ok with the logging.

**Comment by lognaturel** [comment]
Works for me! Will be interesting.

**Pull request merged**